app/frontend/src/cljs/teet/cooperation/cooperation_view.cljs

```diff
@@ -99,9 +99,10 @@
 
 (defn- opinion-status [status]
   (let [status (or status :cooperation.opinion.status/unanswered)]
-    [color-and-status
-     (cooperation-style/opinion-status-color status)
-     (tr-enum status)]))
+    [:div {:data-cy-test (name status)}
```

Comment:
we are using `:data-cy` elsewhere, perhaps we should use the same everywhere for consistency